lib/swaps/Swaps.ts

```diff
@@ -29,14 +31,17 @@ class Swaps extends EventEmitter {
   /** A map between payment hashes and swap deals. */
   private deals = new Map<string, SwapDeal>();
   private usedHashes = new Set<string>();
-  private repository: SwapRepository;
-
+  private swapRepository: SwapRepository;
+  private p2pRepository: P2PRepository;
+  private nodes: NodeList;
   /** The number of satoshis in a bitcoin. */
   private static readonly SATOSHIS_PER_COIN = 100000000;
 
   constructor(private logger: Logger, private models: Models, private pool: Pool, private lndBtcClient: LndClient, private lndLtcClient: LndClient) {
     super();
-    this.repository = new SwapRepository(this.models);
+    this.swapRepository = new SwapRepository(this.models);
+    this.p2pRepository = new P2PRepository(this.models);
+    this.nodes = new NodeList(this.p2pRepository);
```

Comment:
We shouldn't need the list of nodes and p2p repository in the swaps module. The list we load from the database on initialization will not stay in sync as we add new nodes.